docs/source/snakemake/lifecycle.md

```diff
@@ -1,12 +1,12 @@
-# Lifecycle of a Snakemake Execution on Latch
+# Snakemake Execution Lifecycle
 
 Snakemake support is currently based on JIT (Just-In-Time) registration. This means that the workflow produced by `latch register` will register a second workflow, which will run the pipeline tasks. This is because the actual structure of the workflow cannot be specified until parameter values are provided.
```

Comment:
First sentence is confusing for first-time readers. 
 -> "There are two stages to every Snakemake execution: Just-In-Time and runtime ..."